promql/engine.go

```diff
@@ -98,6 +101,8 @@ type query struct {
 	stmt Statement
 	// Timer stats for the query execution.
 	stats *stats.TimerGroup
+	// Result matrix for reuse.
+	mat Matrix
```

Comment:
As part of a struct, I would tend to spell this out.

Reply:
Fixed